ee/desktop/user/server/server.go

```diff
@@ -152,6 +166,63 @@ func (s *UserServer) notificationHandler(w http.ResponseWriter, req *http.Reques
 	w.WriteHeader(http.StatusOK)
 }
 
+func (s *UserServer) showDesktop(w http.ResponseWriter, req *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	s.showDesktopOnceFunc()
+}
+
+type DetectPresenceResponse struct {
+	Success bool   `json:"success"`
+	Error   string `json:"error,omitempty"`
+}
```

Comment:
I'm not convinced on this Response object. Do we really need a bool or can we just rely on error?

Maybe:
```suggestion
type DetectPresenceResponse struct {
	TimeSinceLastDetection string   `json:"time_since_last_detection"`
	Error   string `json:"error,omitempty"`
}
```